rust_dev_preview/dynamodb/README.md

```diff
@@ -1,14 +1,28 @@
-# AWS SDK for Rust code examples for Amazon DynamoDB
+# Rust code examples for DynamoDB
 
-## Purpose
+## Overview
 
-These examples demonstrate how to perform several Amazon DynamoDB (DynamoDB) operations using the developer preview version of the AWS SDK for Rust.
-Most use the schema defined in the __create-table__ example.
```

Comment:
Were lines 5-6 removed unintentionally?

Reply:
The approved template says to use the blurb from the product pages - https://aws.amazon.com/dynamodb/

Reply:
Discussed offline; using both.